prow/cmd/crier/simple_crier_deployment.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: crier
+  namespace: default
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: crier
+  template:
+    metadata:
+      labels:
+        app: crier
+    spec:
+      # Using the service account defined in prow/cluster/pipeline_rbac.yaml
+      serviceAccountName: crier
+      terminationGracePeriodSeconds: 30
+      containers:
+      - name: crier
+        image: gcr.io/k8s-prow/crier:v20190814-9981dc3c5
+        args:
+        - --github-workers=1
+        - --config-path=/etc/config/config.yaml
+        - --github-endpoint=http://ghproxy
+        - --github-endpoint=https://api.github.com
```

Comment:
The instructions you provided don't include setting up ghproxy so the `--github-endpoint` flags will make every request fail on the first attempt. I'd suggest removing them both.